### DISCUSSION TITLE
sparse vector algorithms support ?

### DISCUSSION THREAD

**Discussion by muzhi1991** [meilisearch/product#714] [Feedback & Feature Proposal]
When I looked at the document, I discovered [hybrid search](https://www.meilisearch.com/solutions/vector-search?utm_campaign=vector-search&utm_source=docs). When I looked at this feature in [other vector databases](https://weaviate.io/blog/hybrid-search-explained), I found that it includes dense and sparse indexes. Sparse indexes can use techniques such as SPLADE to improve the effect of semantic similarity.
May I ask if this library supports similar algorithms, or is there any plan to support it?
Thank you for your great work！

**Comment by Kerollmops** [comment]
Hey @muzhi1991 👋

The current version of the Hybrid Search of Meilisearch doesn't use sparse vectors.

First, because we didn't worked on it and second because it is a technic mostly used to do full text search and Meilisearch is already one of the most proficient keyword search engine on the market.

We will evaluate the need of sparse vectors in the future.

**Reply by muzhi1991** [reply]
Thank you for your reply. Full text search is great. For the hybrid search feature, sparse vectors can make it more customized.
